jujugui/static/gui/src/app/store/env/api.js

```diff
@@ -2056,7 +2074,8 @@ window.yui.add('juju-env-api', function(Y) {
         collected[key] = '';
         return collected;
       }, {});
-      this.update_annotations(entity, type, data, callback);
+      this.update_annotations(entity, type, data, callback,
```

Comment:
Should we make this get options as well, and then call `this.update_annotations(entity, type, data, callback, options);`?

Reply:
Makes sense. I was trying to be too smart and guess where we'd use this in the future, but this is for the best.